frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -335,3 +335,152 @@ export function PageHeaderCustom(): JSX.Element {
+                <Link className="font-semibold" to="https://posthog.com/docs/experiments/testing-and-launching">
+                    test your feature flag
+                </Link>
+                . Once everything works as expected, you can launch your experiment. From that point, any new experiment
+                events will be counted towards the results.
+            </LemonBanner>
+        )
+    }
+
+    // Running, results present, not significant
+    if (isExperimentRunning && experimentResults && !isExperimentStopped && !areResultsSignificant) {
+        // Results insignificant, but a large enough sample/running time has been achieved
+        // Further collection unlikely to change the result -> recommmend cutting the losses
+        if (
+            experimentInsightType === InsightType.FUNNELS &&
+            funnelResultsPersonsTotal > Math.min(recommendedSampleSize, 500)
+        ) {
+            return (
+                <LemonBanner type="warning" className="mt-4">
+                    You've reached a robust sample size for your experiment, but the results are still inconclusive.
+                    Continuing the experiment is unlikely to yield significant findings. It may be time to stop this
+                    experiment.
+                </LemonBanner>
+            )
+        }
+        if (experimentInsightType === InsightType.TRENDS && actualRunningTime > Math.min(recommendedRunningTime, 7)) {
+            return (
+                <LemonBanner type="warning" className="mt-4">
+                    Your experiment has been running long enough, but the results are still inconclusive. Continuing the
+                    experiment is unlikely to yield significant findings. It may be time to stop this experiment.
+                </LemonBanner>
+            )
+        }
+
+        return (
+            <LemonBanner type="info" className="mt-4">
+                Your experiment is live and is collecting data, but hasn't yet reached the statistical significance
+                needed to make reliable decisions. It's important to wait for more data to avoid premature conclusions.
+            </LemonBanner>
+        )
+    }
+
+    // Running, results significant
+    if (isExperimentRunning && !isExperimentStopped && areResultsSignificant && experimentResults) {
+        const { probability } = experimentResults
+        const winningVariant = getHighestProbabilityVariant(experimentResults)
+        if (!winningVariant) {
+            return <></>
+        }
+
+        const winProbability = probability[winningVariant]
+
+        // Win probability only slightly over 0.9 and the recommended sample/time just met -> proceed with caution
+        if (
+            experimentInsightType === InsightType.FUNNELS &&
+            funnelResultsPersonsTotal > Math.min(recommendedSampleSize) + 50 &&
+            winProbability < 0.91
```

Comment:
Somewhat arbitrary, feel free to adjust

```suggestion
            winProbability < 0.93
```